SDK/MobileCenterDistribute/Microsoft.Azure.Mobile.Distribute.iOS/Distribute.cs

```diff
@@ -8,7 +8,19 @@ namespace Microsoft.Azure.Mobile.Distribute
     public static partial class Distribute
     {
         [Preserve]
-        public static Type BindingType => typeof(iOSDistribute);
+        public static Type BindingType
+        {
+            get
+            {
+
+// We cannot use Conditional Attributes as they only work for non-void methods and classes, not for properties.
+#if DEBUG
```

Comment:
This will not work on consumer side (once in nuget), the sdk is always shipped in release, same issue as ios native.

When you build locally forms puppet you build both sdk and app to debug that's why it worked for you, but in demo debug with nuget it will not work.